ApsimNG/Utility/FindAndReplaceForm.cs

```diff
@@ -69,17 +72,37 @@ private void Window1_Destroyed(object sender, EventArgs e)
 
         public void Destroy()
         {
+#if NETCOREAPP
+            window1.Dispose();
+#else
             window1.Destroy();
+#endif
         }
 
         private void Window1_DeleteEvent(object o, DeleteEventArgs args)
         {
+#if NETCOREAPP
+            context.Highlight = false;
+#endif
             window1.Hide();
             args.RetVal = true;
         }
 
+#if NETCOREAPP
+        SearchContext context;
+        SearchContext Context
+        {
+            get { return context; }
+            set { context = value; }
+        }
+
+        SourceView editor;
+        SourceView Editor
```

Comment:
Could we have some explicit accessibility modifiers here? We also probably don't need to have a manual backing field for the Context property.

```suggestion
#if NETCOREAPP
        public SearchContext Context { get; set; }

        private SourceView editor;
        public SourceView Editor
```